app/assets/stylesheets/application.scss

```diff
@@ -591,21 +947,174 @@ p.adventure-link a {
 }
 
 
-// new bucket list
+// Button styles //
 
-input.button.save-button {
-  margin-top: 35px;
-  margin-bottom: 15px;
+.tiny-button {
+  display: inline-block;
+  width: 100%;
+  color: $oil;
+  text-align: right;
+  padding: .6rem 0rem .1rem 0rem;
+  margin: 0;
+  font-size: 1rem;
+  height: 50px;
+
+  &:hover p {
+    display: inline-block;
+    width: 100%;
+    color: $oil;
+    text-align: right;
+    font-size: 1rem;
+    padding: 0;
+    margin: 0;
+    height: 50px;
+    transition: all .25 ease-in-out;
+  }
+
+  &:hover a {
+    display: inline-block;
+    width: 100%;
+    color: $third-color;
+    text-align: right;
+    font-size: 1rem;
+    padding: 0;
+    margin: 0;
+    height: 50px;
+    transition: all .25 ease-in-out;
+  }
+}
+
+.button {
+  padding: .5rem 1rem .5rem 1rem;
+  text-align: center;
+  border-width: 1px;
+  border-color: $base;
+  color: $oil;
+  font-weight: bold;
+
+  &:hover {
+    background-color: $third-color;
+    color: $oil;
+    font-weight: bold;
+    transition: all .25 ease-in-out;
+  }
+}
+
+.small-button {
+  text-align: center;
+  font-weight: bold;
+  color: $oil;
+  padding: .5rem;
+  background-color: $small-button-color;
+  border: 1px solid $base;
+
+  &:hover {
+    background-color: $secondary-transparent-color;
+    color: $oil;
+    transition: all .25 ease-in-out;
+  }
+}
+
+.delete a {
+  color: $oil;
+  padding: .5rem;
+  background: rgba(250,250,250,0.7);
+  border: 1px solid $base;
+
+  &:hover {
+    background-color: $secondary-transparent-color;
+    transition: all .25 ease-in-out;
+  }
+}
+
+.delete {
+  margin-right: 4%;
+}
+
+.cancel {
+  color: $oil;
+  background-color: $small-button-color;
+  margin-top: 15px;
+  margin-bottom: 0;
+
+  a {
+    color: inherit;
+  }
+
+  &:hover {
+    color: inherit;
+    background-color: $secondary-transparent-color;
+    transition: all .25 ease-in-out;
+  }
+}
+
+.new-save-button {
+  margin-left: 41%;
```

Comment:
Properties should be ordered margin-bottom, margin-left, margin-right, margin-top